test/markup/latex/verbatim.expect.txt

```diff
@@ -0,0 +1,48 @@
+<span class="hljs-keyword">\verb</span>|<span class="hljs-string">\verb</span>|<span class="hljs-keyword">\verb</span>!<span class="hljs-string">%\&amp;$^_{}</span>!foo
+<span class="hljs-keyword">\verb</span>{<span class="hljs-string">\verb</span>{<span class="hljs-keyword">\verb</span>}<span class="hljs-string">\verb{</span>}
```

Comment:
This is correct because verb can only be VERBATIM_DELIMITED_EQUAL so the delimiters here are `{` on both sides?

Reply:
Precisely.